test/linalg/lq.jl

```diff
@@ -207,14 +207,12 @@ end
     # where the LQ-factored matrix has more columns n than rows m,
     # Q's square form is n-by-n whereas its thin form is m-by-n.
     # hence we need also test *-by-m (m-by-*) C with
-    # A_mul_B*(C, Q) (Ac_mul_B*(C, Q)) ops, as below via m-by-m C.
+    # A*_mul_B(C, Q) ops, as below via m-by-m C.
     mA, nA = 3, 4
     implicitQ, explicitQ = getqs(lqfact(randn(mA, nA)))
     C = randn(mA, mA)
     zeroextCright = hcat(C, zeros(eltype(C), mA))
     zeroextCdown = vcat(C, zeros(eltype(C), (1, mA)))
     @test *(C, implicitQ) ≈ *(zeroextCright, explicitQ)
-    @test A_mul_Bc(C, implicitQ) ≈ A_mul_Bc(zeroextCright, explicitQ)
     @test Ac_mul_B(C, implicitQ) ≈ Ac_mul_B(zeroextCdown, explicitQ)
-    @test Ac_mul_Bc(C, implicitQ) ≈ Ac_mul_Bc(zeroextCdown, explicitQ)
```

Comment:
This only deletes tests – are there any tests that could be added that would have caught this?

Reply:
Excellent point, and yes! :) I've added `@test_throws` making certain those shape combinations now yield `DimensionMismatch`s. Thanks!